### DISCUSSION TITLE
Retrieve ip address using CLI

### DISCUSSION THREAD

**Discussion by igorwojda** [ProxymanApp/Proxyman#2150] [Q&A]
Is it possible to get in address ProxyMan is listening on using `proxyman-cli` ?
<img width="493" alt="image" src="https://github.com/user-attachments/assets/3592ccaa-d5cb-46e0-8bcf-0e2884d2a888">

Use case:
I have Android device where I am configuring proxy using adb.
Occasionally IP address of my computer changes, so I would like to get current IP address Proxyman is listening on from `proxyman-cli` (and pass it to adb) rather than depending on hardcoded value `192.168.1.15`

#Configure Android Proxy
# Enable Android proxy
`alias adb-proxy-enable='adb shell settings put global http_proxy 192.168.1.15:9090 && adb shell settings put global global_http_proxy_host 192.168.1.15 && adb shell settings put global global_http_proxy_port 9090'`

# Disable Android proxy
`alias adb-proxy-disable='adb shell settings delete global http_proxy && adb shell settings delete global global_http_proxy_host && adb shell settings delete global global_http_proxy_port && adb shell settings put global http_proxy :0'`

**Comment by NghiaTranUIT** [comment]
It's not available yet, but you can simply retrieve the current IPv4 that Proxyman is listening to with this command: 
```sh
ifconfig | grep "inet " | grep -v 127.0.0.1 | awk '{print $2}'
```
